script/tool/lib/src/fetch_deps_command.dart

```diff
@@ -63,12 +169,81 @@ class FetchDepsCommand extends PackageLoopingCommand {
 
       final String packageName = package.directory.basename;
 
-      final int exitCode = await gradleProject.runCommand('$packageName:dependencies');
+      final int exitCode =
+          await gradleProject.runCommand('$packageName:dependencies');
       if (exitCode != 0) {
         return PackageResult.fail();
       }
     }
 
     return PackageResult.success();
   }
+
+  Future<PackageResult> _fetchDarwinDeps(
+      RepositoryPackage package, final String platform) async {
+    if (!pluginSupportsPlatform(platform, package,
+        requiredMode: PlatformSupport.inline)) {
+      final String displayPlatform = platform.replaceFirst('os', 'OS');
+      return PackageResult.skip(
+          'Package does not have native $displayPlatform dependencies.');
+    }
+
+    for (final RepositoryPackage example in package.getExamples()) {
+      final Directory platformDir =
+          example.platformDirectory(getPlatformByName(platform));
+
+      final File generatedXCConfig = platform == platformMacOS
+          ? platformDir
+              .childDirectory('Flutter')
+              .childDirectory('ephemeral')
+              .childFile('Flutter-Generated.xcconfig')
+          : platformDir
+              .childDirectory('Flutter')
+              .childFile('Generated.xcconfig');
+      // Running `pod install` requires `flutter pub get` or `flutter build` to
```

Comment:
FWIW I think you can --config-only for ios builds. That said I think pub get is probably the more stable command to run here.

Reply:
Added.